code/datums/mapgen/planetary/WasteGenerator.dm

```diff
@@ -89,6 +89,7 @@
 /datum/biome/waste
 	open_turf_types = list(
 		/turf/open/floor/plating/asteroid/wasteplanet/lit = 80,
+		/turf/open/floor/plating/rust/wasteplanet/lit = 15,
```

Comment:
Why are you adding this in? It seems to be defined below already.

Reply:
I don't think I added that. This PR was made before the biomes were merged so any pulls from that probably got dragged into this.

Reply:
It's probably left over but it does need to go.

Reply:
Fucky Wucky Obliterated !